master/reference/advanced/etcd-rbac/calico-etcdv3-paths.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Calico key and path prefixes in etcd v3
+---
+
+The Paths listed here are the key or path prefixes that a particular {{site.prodname}}
+component needs access to in etcd to function successfully.
+
+> **Note**: The path prefixes listed here may change in the future and at that point anything
+> referencing them (like etcd roles) would need to be updated appropriately.
+{: .alert .alert-info}
+
+
+## calico/node
+
+| Path                                      | Access |
+|-------------------------------------------|--------|
+| /calico/felix/v1/\*                       |   RW   |
+| /calico/ipam/v2/\*                        |   RW   |
+| /calico/resources/v3/projectcalico.org/\* |   RW   |
+
+## felix as a stand alone process
+
+| Path                                      | Access |
+|-------------------------------------------|--------|
+| /calico/felix/v1/\*                       |   RW   |
+| /calico/resources/v3/projectcalico.org/\* |   RW   |
+
+## CNI-plugin
+
+| Path                                      | Access |
+|-------------------------------------------|--------|
+| /calico/ipam/v2/\*                        |   RW   |
+| /calico/resources/v3/projectcalico.org/\* |   RW   |
+
+## calico/kube-controllers
+
+| Path                                      | Access |
+|-------------------------------------------|--------|
+| /calico/resources/v3/projectcalico.org/\* |   RW   |
```

Comment:
This will need the IPAM tree as well, since the node controller releases IP addresses.

Reply:
ah yes it does.  ta!